app/lib/tabs/search-tab.dart

```diff
@@ -0,0 +1,124 @@
+import 'package:bloc_pattern/bloc_pattern.dart';
+import 'package:flutter/material.dart';
+//TODO: When do we import index vs file itself
+import 'package:inside_chassidus/routes/player-route/index.dart';
+import 'package:inside_chassidus/routes/search-section-route.dart';
+import 'package:inside_chassidus/tabs/widgets/simple-media-list-widgets.dart';
+import 'package:inside_chassidus/util/search/search-service.dart';
+import 'package:inside_data_flutter/inside_data_flutter.dart';
+
+//TODO: Break this down into smaller widgets and add necessary navigator stuff; see MediaListTabNavigator
```

Comment:
Is your preference to use methods unless otherwise necessary? I've personally started using classes a lot more often as I've come across different areas where methods lack in comparison to classes.

Reply:
Pretty much I prefer methods.
A class for a simple piece of UI can make things more difficult to follow.
I tend to make a class for "something complicated enough that I don't want to think about it right now"
But outside of the extremes (an enormous or trivial widget), its fine either way